modules/base.py

```diff
@@ -194,6 +195,17 @@ class K8sServiceModuleProcessor(ModuleProcessor):
     # TODO(patrick): Remove this flag and references to it once all clouds support multiple ports
     FLAG_MULTIPLE_PORTS_SUPPORTED = False
 
+    def pre_hook(self, module_idx: int) -> None:
+        pending_upgrade_helm_chart = Helm.get_helm_list(
+            release=f"{self.layer.name}-{self.module.name}"
+        )  # , status="pending-upgrade")
+        if pending_upgrade_helm_chart:
+            raise UserErrors(
+                f"There is a pending upgrade for the helm chart: {self.layer.name}-{self.module.name}."
+                "\nPlease use `opta force-unlock` to rollback already existing module."
+            )
+        return super(K8sServiceModuleProcessor, self).pre_hook(module_idx)
```

Comment:
```suggestion
        return super().pre_hook(module_idx)
```

Use of the short form of `super` is preferred in Python 3.